google/resource_sql_database.go

```diff
@@ -62,6 +75,14 @@ func resourceSqlDatabaseCreate(d *schema.ResourceData, meta interface{}) error {
 		Instance: instance_name,
 	}
 
+	if v, ok := d.GetOk("charset"); ok {
```

Comment:
Since we have a default set, we should always get a value from this; we can change both values from `d.GetOk` to `d.Get`. We can also inline these values into the struct initializer of `db`.